x-pack/plugins/ml/public/application/routing/routes/datavisualizer/index_based.tsx

```diff
@@ -43,19 +36,15 @@ export const indexBasedRouteFactory = (
   ],
 });
 
-const PageWrapper: FC<PageProps> = ({ location, deps }) => {
-  const { redirectToMlAccessDeniedPage } = deps;
-
-  const { index, savedSearchId }: Record<string, any> = parse(location.search, { sort: false });
-  const { context } = useResolver(index, savedSearchId, deps.config, deps.dataViewsContract, {
-    checkBasicLicense,
-    cacheDataViewsContract: () => cacheDataViewsContract(deps.dataViewsContract),
-    checkGetJobsCapabilities: () => checkGetJobsCapabilitiesResolver(redirectToMlAccessDeniedPage),
-  });
+const PageWrapper: FC = () => {
+  // TODO check if required capabilities is correct
```

Comment:
Can this be removed before merging assuming all tests are passing?

Reply:
removed in https://github.com/elastic/kibana/pull/159176/commits/71565c45c0401e2d44be5d03a934f8c34c1403f1